.rustfmt.toml

```diff
@@ -1,2 +1,5 @@
 hard_tabs = true
 match_block_trailing_comma = true
+
+unstable_features = true
```

Comment:
I don't think we should do it this way -- instead, refer to the rustls workflow I linked to in your other PR which has two separate rustfmt configs and makes sure that the default config can still run with stable rustfmt.

Reply:
Done.

I've also added `imports_granularity = "Module"` since that was in the linked workflow.

> It is not clear to me why the github ui is expecting a Format & clippy job.

This also appears resolved (note that I also renamed the workflow job from `fmt` to `fmt-unstable`)